lib/pos_blockchain.go

```diff
@@ -276,13 +276,14 @@ func (bc *Blockchain) processBlockPoS(block *MsgDeSoBlock, currentView uint64, v
 
 	// We expect the utxoView for the parent block to be valid because we check that all ancestor blocks have
 	// been validated.
-	parentUtxoView, err := bc.getUtxoViewAtBlockHash(*block.Header.PrevBlockHash)
+	parentUtxoViewAndUtxoOps, err := bc.getUtxoViewAtBlockHash(*block.Header.PrevBlockHash)
```

Comment:
May as well rename the function too while at it
```suggestion
	parentUtxoViewAndUtxoOps, err := bc.getUtxoViewAndUtxoOpsAtBlockHash(*block.Header.PrevBlockHash)
```